src/main/java/com/github/devotedmc/hiddenore/tracking/BreakTracking.java

```diff
@@ -40,54 +30,61 @@ public BreakTracking() {
-		} else {
-			try {
-				DataInputStream dis = new DataInputStream(new BufferedInputStream(new FileInputStream(tf)));
-
-				String uuid = null;
-				boolean active = true;
-				while (active) {
-					try {
-						uuid = dis.readUTF();
-						HiddenOre.getPlugin().getLogger().info("Loading tracking data for world " + uuid);
-					} catch (EOFException done) {
-						active = false;
-						break;
-					}
-					UUID uid = UUID.fromString(uuid);
-					Map<Long, short[]> world = new HashMap<Long, short[]>();
-					track.put(uid, world);
-
-					long ccnt = 0l;
 
-					while (dis.readBoolean()) {
-						Long chunk = dis.readLong();
-						short[] layers = new short[256];
-						for (int i = 0; i < layers.length; i++) {
-							layers[i] = dis.readShort();
+		try {
+			long s = System.currentTimeMillis();
+			HiddenOre.getPlugin().getLogger().info("Starting Break Tracking load");
+			File tf = Config.getTrackFile();
+			if (!tf.exists()) {
+				HiddenOre.getPlugin().getLogger().info("No save exists to load");
+			} else {
+				try {
+					DataInputStream dis = new DataInputStream(new BufferedInputStream(new FileInputStream(tf)));
+
+					String uuid = null;
+					boolean active = true;
+					while (active) {
+						try {
+							uuid = dis.readUTF();
+							HiddenOre.getPlugin().getLogger().info("Loading tracking data for world " + uuid);
+						} catch (EOFException done) {
+							active = false;
+							break;
 						}
-						if (Config.isDebug) {
-							HiddenOre.getPlugin().getLogger().info("Loaded layers for chunk " + chunk);
+						UUID uid = UUID.fromString(uuid);
+						Map<Long, short[]> world = new HashMap<Long, short[]>();
+						track.put(uid, world);
+
+						long ccnt = 0l;
+
+						while (dis.readBoolean()) {
+							Long chunk = dis.readLong();
+							short[] layers = new short[256];
+							for (int i = 0; i < layers.length; i++) {
+								layers[i] = dis.readShort();
+							}
+							if (Config.isDebug) {
+								HiddenOre.getPlugin().getLogger().info("Loaded layers for chunk " + chunk);
+							}
+							ccnt++;
+							world.put(chunk, layers);
 						}
-						ccnt++;
-						world.put(chunk, layers);
+
+						HiddenOre.getPlugin().getLogger().info("Loaded " + ccnt + " chunks");
 					}
 
-					HiddenOre.getPlugin().getLogger().info("Loaded " + ccnt + " chunks");
+					dis.close();
+				} catch (IOException ioe) {
+					HiddenOre.getPlugin().getLogger().log(Level.SEVERE, "Failed to load break tracking.", ioe);
 				}
-
-				dis.close();
-			} catch (IOException ioe) {
-				HiddenOre.getPlugin().getLogger().log(Level.SEVERE, "Failed to load break tracking.", ioe);
 			}
+			s = System.currentTimeMillis() - s;
+			HiddenOre.getPlugin().getLogger().info("Took " + s + "ms to load Break Tracking");
+		}
+		catch(Exception e) {
+			HiddenOre.getPlugin().getLogger().warning("Error enabling break tracking. Stacktrace:");
+			HiddenOre.getPlugin().getLogger().warning(e.getLocalizedMessage());
```

Comment:
Generally better here to use the baked in error formatter, like so:

HiddenOre.getPlugin().getLogger().warning("Error enabling break tracking. Stacktrace:", e);